### PR TITLE
Update README with info to get and build code.

### PR DESCRIPTION
**Pull request by acalloo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,10 @@ All these abstractions are handled at compilation using C++ template meta-progra
 If you like the project, please leave us a github star.
 
 If you want to know more, join un on [Slack](https://join.slack.com/t/ddc-lib/shared_invite/zt-14b6rjcrn-AwSfM6_arEamAKk_VgQPhg)
+
+## Getting the code and basic installation
+git clone --recurse-submodules -j8 https://github.com/CExA-project/ddc.git
+
+cmake -B build -D DDC_BUILD_KERNELS_FFT=OFF -D DDC_BUILD_KERNELS_SPLINES=OFF -D DDC_BUILD_PDI_WRAPPER=OFF
+
+cmake --install install
```
Comment (tpadioleau) [inline README.md:34]: Installation is kind of broken for now, i would strongly recommend to only build with `--build build`

File: README.md [hunk 2]
```diff
@@ -25,3 +25,10 @@ All these abstractions are handled at compilation using C++ template meta-progra
 If you like the project, please leave us a github star.
 
 If you want to know more, join un on [Slack](https://join.slack.com/t/ddc-lib/shared_invite/zt-14b6rjcrn-AwSfM6_arEamAKk_VgQPhg)
+
+## Getting the code and basic installation
+git clone --recurse-submodules -j8 https://github.com/CExA-project/ddc.git
+
+cmake -B build -D DDC_BUILD_KERNELS_FFT=OFF -D DDC_BUILD_KERNELS_SPLINES=OFF -D DDC_BUILD_PDI_WRAPPER=OFF
```
Comment (tpadioleau) [inline README.md:32]: Should we add `cd ddc` before ?

File: README.md [hunk 3]
```diff
@@ -25,3 +25,10 @@ All these abstractions are handled at compilation using C++ template meta-progra
 If you like the project, please leave us a github star.
 
 If you want to know more, join un on [Slack](https://join.slack.com/t/ddc-lib/shared_invite/zt-14b6rjcrn-AwSfM6_arEamAKk_VgQPhg)
+
+## Getting the code and basic installation
+git clone --recurse-submodules -j8 https://github.com/CExA-project/ddc.git
```
Comment (tpadioleau) [inline README.md:30]: I did not know about this `-j` option in git :)

**Review by tpadioleau — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (acalloo) [reply README.md:34]: Got that, I didn't try!

File: README.md (hunk 2 above)
Comment (acalloo) [reply README.md:32]: I'll add that line before the rest.

**Pull request merged**